#include <iostream>
using namespace std;
int main()
{
  string s;
  getline(cin,s);
  cout<<"Start\nInside try block.\nCaught an exception value is="<<s<<"\nEnd";
  return 0;
}